.github/workflows/release-manual-trigger.yml

```
@@ -231,5 +243,66 @@ jobs:
           echo "  Release URL: ${{ steps.gh-release.outputs.url }}"
           echo "  Download URL: ${{ fromJSON(steps.gh-release.outputs.assets)[0].browser_download_url }}"
 
+  update-cask-definition:
+    # It needs manual-build, but wait to update the cask until the release job succeeds.
+    needs:
+      - manual-build
+      - manual-release
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE_VERSION: "${{ needs.manual-build.outputs.extracted-package-version }}"
+      SHA256: "${{ needs.manual-build.outputs.dmg-sha256 }}"
+      CASK_FILE_PATH: "Casks/pikachuexe-freetube.rb"
+      CASK: "pikachuexe-freetube"
+
+    steps:
+      - name: Checkout Tap Repository
+        uses: actions/checkout@v4
+
+      - name: Update Cask Definition File
+        id: update_cask_file
+        continue-on-error: true
+        run: |
+          sed -i '' \
+            -e "s/version \"[^\"]*\"/version \"$PACKAGE_VERSION\"/" \
+            -e "s/sha256 \"[^\"]*\"/sha256 \"$SHA256\"/" \
+            "${{ env.CASK_FILE_PATH }}"
+
+      - name: Get Token
+        if: steps.update_cask_file.outcome == 'success'
+        id: get_workflow_token
+        uses: peter-murray/workflow-application-token-action@v3
+        with:
+          application_id: ${{ secrets.APPLICATION_ID }}
+          application_private_key: ${{ secrets.APPLICATION_PRIVATE_KEY }}
+          revoke_token: true
+          permissions: "contents:write, metadata:read, pull_requests:write"
+
+      - name: Create Pull Request
+        if: steps.update_cask_file.outcome == 'success'
+        id: create-pr
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: "${{ steps.get_workflow_token.outputs.token }}"
+          commit-message: "$CASK $PACKAGE_VERSION"
+          delete-branch: true
```

Is this the right way to get separate branch per package version?

```suggestion
          branch: "update/cask-file/$PACKAGE_VERSION"
          delete-branch: true
```

---

I think so. If you want to use the branch naming convention that brew uses it would be: `"bump-$CASK-$PACKAGE_VERSION"`